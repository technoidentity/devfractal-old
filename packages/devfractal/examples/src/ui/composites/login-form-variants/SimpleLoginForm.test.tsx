import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { SimpleLoginForm } from './SimpleLoginForm'

test('Simple Login Form', () => {
  const component: ReactTestRenderer = renderer.create(<SimpleLoginForm />)
  expect(component.toJSON()).toMatchSnapshot()
})
