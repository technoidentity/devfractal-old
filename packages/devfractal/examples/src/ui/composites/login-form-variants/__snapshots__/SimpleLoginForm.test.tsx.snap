// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Login Form 1`] = `
<section
  className="section"
>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label
        className="label"
        htmlFor="username"
      >
        Username:
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          id="username"
          name="username"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="label"
        htmlFor="password"
      >
        Password:
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          id="password"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
    <div
      className="buttons"
    >
      <button
        className="button is-info"
        disabled={false}
        type="submit"
      >
        Submit
      </button>
      <button
        className="button is-danger"
        disabled={true}
        onClick={[Function]}
        type="reset"
      >
        Reset
      </button>
    </div>
    <pre>
      {
  "username": "",
  "password": ""
}
    </pre>
  </form>
</section>
`;
